xtos/include/sof/lib/dma.h

```diff
@@ -105,8 +105,15 @@ enum dma_irq_cmd {
 #define DMA_ATTR_COPY_ALIGNMENT			1
 #define DMA_ATTR_BUFFER_ADDRESS_ALIGNMENT	2
 #define DMA_ATTR_BUFFER_PERIOD_COUNT		3
+#else
+/* TGL platform with Zephyrs native drivers still build some dma drivers from SOF.
```

Comment:
instead of temporary workaround we can simply remove dma files from tgl zephyr cmakelists, it is not needed